### PR TITLE
Localize dotnet-remove-p2p

### PR DESCRIPTION
**Pull request by vsccarl**
This was written after we started localizing and has references to a non-existent LocalizableStrings file, such as LocalizableStrings.RequiredArgumentNotPassed. I'm not sure what the text is for these strings.

### CODE REVIEW

**Comment by piotrpMSFT** [comment]
@MattGertz can you pre-approve this change so we can merge as soon as it is green?

**Comment by piotrpMSFT** [comment]
@vsccarl this PR is failing with `error CS0117: 'LocalizableStrings' does not contain a definition for 'RequiredArgumentNotPassed'`. Can you address?

**Comment by nguerrera** [comment]
Looks like there are 2 classes named LocalizableStrings involved:

* Microsoft.DotNet.Tools.Remove.ProjectToProjectReference.LocalizableStrings
* Microsoft.DotNet.Tools.LocalizableStrings

The first one is new in this PR and the second is pre-existing. The new one is shadowing the existing one and causing the reference to the existing one's RequiredArgumentNotPassed to fail to bind. 

Probably the second one should be called CommonLocalizableStrings or something. Otherwise its usage will need namespace qualification.

**Comment by piotrpMSFT** [comment]
Thanks Nick!

**Comment by MattGertz** [comment]
Approved pending tests passing.

File: src/dotnet/commands/dotnet-remove-p2p/Program.cs [hunk 1]
```diff
@@ -16,29 +16,27 @@ public static int Run(string[] args)
...
-                "The project file to modify. If a project file is not specified," +
-                " it searches the current working directory for an MSBuild file that has" +
-                " a file extension that ends in `proj` and uses that file.");
+                $"<{LocalizableStrings.CmdArgProject}>",
+                LocalizableStrings.CmdArgumentDescription);
 
             CommandOption frameworkOption = app.Option(
-                "-f|--framework <FRAMEWORK>",
-                "Remove reference only when targetting a specific framework",
+                $"-f|--framework <{LocalizableStrings.CmdFramework}>",
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-remove-p2p/Program.cs]: Again, not sure what Cmd means here.

**Comment by piotrpMSFT** [comment]
Rebased atop #4917

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Ubuntu x64 Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Ubuntu16.04 x64 Debug Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Ubuntu x64 Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Ubuntu16.04 x64 Debug Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OSX x64 Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OSX x64 Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OSX x64 Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OSX x64 Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OSX x64 Release Build please.

**Pull request merged**